<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>13</y>
      <width>671</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Очистить</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Имя</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nameEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Номер</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="phoneEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="text">
            <string>Добавить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editButton">
           <property name="text">
            <string>Изменить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeButton">
           <property name="text">
            <string>Удалить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="findButton">
           <property name="text">
            <string>Найти</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="saveAction"/>
    <addaction name="loadAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="mainHelpAction"/>
    <addaction name="loadHelpAction"/>
    <addaction name="authorsHelpAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="helpAction">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="loadAction">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="mainHelpAction">
   <property name="text">
    <string>Основные функции</string>
   </property>
  </action>
  <action name="loadHelpAction">
   <property name="text">
    <string>Сохранение и загрузка</string>
   </property>
  </action>
  <action name="authorsHelpAction">
   <property name="text">
    <string>Авторы</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
